indico/modules/events/contributions/controllers.py

```diff
@@ -95,8 +78,38 @@ class RHContributions(RHManageContributionsBase):
     """Display contributions management page"""
 
     def _process(self):
+        report_config = self.reporter.get_config()
+        if 'config' in request.args:
```

Comment:
The problem is that the only clean way to redirect is to return it from `_process` or `_checkParams` in the RH. While you could raise a HTTPException subclass with a redirect response attached, that's not really clean since it's not an *exceptional* case.